build.gradle.kts

```diff
@@ -1,17 +1,12 @@
 /*
- * Copyright 2023 Responsive Computing, Inc.
+ * Copyright 2024 Responsive Computing, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * This source code is licensed under the Responsive Software License Agreement v1.0 available at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * https://www.responsive.dev/licenses/rbsl10
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * This software requires a valid Commercial License Key for production use. Trial and commercial
+ * licenses can be obtained at https://www.responsive.dev/sdk/get-started.
```

Comment:
Making sure that this is the legit URL. Or if we want to keep the option to move things around later, lets just say 'go to https://www.responsive.dev'.  I'd recommend the latter.